.github/workflows/cli_commands.yml

```diff
@@ -0,0 +1,250 @@
+                  echo "packed_file_absolute_path=${TARGET_PACKED_FILE_PATH}" >> $GITHUB_OUTPUT
+
+            - name: Create external testing environment
+              id: create_test_environment
+              run: |
+                  # Create a fresh test directory to simulate external user environment
+                  EXTERNAL_TEST_DIR="${{ github.workspace }}/external-test-env"
+                  mkdir -p "${EXTERNAL_TEST_DIR}"
+                  echo "external_test_directory=${EXTERNAL_TEST_DIR}" >> $GITHUB_OUTPUT
+
+            - name: Install packed azle in external environment
+              working-directory: ${{ steps.create_test_environment.outputs.external_test_directory }}
+              run: |
+                  # Path to the packed azle.tgz file
+                  AZLE_PACKAGE="${{ steps.pack_azle_source.outputs.packed_file_absolute_path }}"
+
+                  # Install the package locally for testing
+                  echo "Installing the packed azle package for CLI testing"
+                  npm install "${AZLE_PACKAGE}"
+
+            - name: Test azle version command
+              working-directory: ${{ steps.create_test_environment.outputs.external_test_directory }}
+              run: |
+                  echo "Testing 'azle --version' command"
+                  DISPLAYED_VERSION=$(npx azle --version)
+                  echo "Displayed version: ${DISPLAYED_VERSION}"
+
+                  # Verify the version matches our unique test version
+                  if [[ "${DISPLAYED_VERSION}" == *"${UNIQUE_VERSION}"* ]]; then
+                    echo "✅ Version verification successful: Running the correct packed version"
+                  else
+                    echo "❌ Version verification failed: Version mismatch"
+                    echo "Expected to find: ${UNIQUE_VERSION}"
+                    echo "But got: ${DISPLAYED_VERSION}"
+                    exit 1
+                  fi
+
+            - name: Test azle dev setup command
+              working-directory: ${{ steps.create_test_environment.outputs.external_test_directory }}
+              run: |
+                  echo "Testing 'azle dev setup' command"
+                  npx azle dev setup
+
+            - name: Test azle dev template commands
+              working-directory: azle
+              run: |
+                  echo "Testing azle dev template commands"
+
+                  # Remove existing template files to test regeneration
+                  echo "Removing existing template files for regeneration test"
+                  rm -f dist/canister_templates/stable.wasm
+                  rm -f dist/canister_templates/experimental.wasm
+
+                  # Test azle dev template (stable)
+                  echo "Testing 'azle dev template' command"
+                  npx azle dev template
+
+                  # Verify stable template was created
+                  if [[ -f "dist/canister_templates/stable.wasm" ]]; then
```

Comment:
Also check that the wasm file lengths are not 0

Reply:
Done